Source/Atomic/Graphics/Direct3D9/D3D9Graphics.cpp

```diff
@@ -268,6 +268,7 @@ Graphics::Graphics(Context* context) :
     instancingSupport_(false),
     sRGBSupport_(false),
     sRGBWriteSupport_(false),
+    numSinglePassPrimitives_(0),

```

Comment:
numPasses_ should be initialized here as well, even if it is first Reset before used